roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -77,9 +76,14 @@
   - name: "Process support template"
     shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig process logging-support-template |  {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f -"
 
+  - name: "Process imagestream template"
+    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig process logging-imagestream-template -v {{ oc_process_values }} |  {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f -"
```

Comment:
looks like `oc process KEY1=VALUE1 KEY2=VALUE2` works too so maybe just `{{ ip_kv }} {{ iv_kv }}` and then get rid of oc_process_values all together so we don't have to deal with whether or not to use -v